OpenSOC-Alerts/src/main/java/com/opensoc/alerts/adapters/FalconHoseAlertAdapter.java

```diff
@@ -0,0 +1,75 @@
+package com.opensoc.alerts.adapters;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.json.simple.JSONObject;
+import org.apache.log4j.Logger;
+import com.opensoc.alerts.interfaces.AlertsAdapter;
+
+public class FalconHoseAlertAdapter implements AlertsAdapter, Serializable {
+
+	protected static final Logger LOG = Logger
+			.getLogger(FalconHoseAlertAdapter.class);
+
+    public FalconHoseAlertAdapter(Map<String, String> config) {
+    }
+
+    @Override
+    public boolean initialize() {
+        return true;
+    }
+
+    @Override
+    public boolean refresh() throws Exception {
+        return false;
+    }
+
+    @Override
+    public Map<String, JSONObject> alert(JSONObject raw_message) {
+        Map<String, JSONObject> alerts = new HashMap<String, JSONObject>();
+
+        JSONObject content = (JSONObject)raw_message.get("message");
+
+        String source = "unknown";
+        String dest = "unknown";
+
+        if (content.containsKey("ip_src_addr")) {
+            source = content.get("ip_src_addr").toString();
```

Comment:
Yeah, I didn't know whether the `designated_host` should be the source or dest IP. (The other alert adapters look up a "whitelist" in HBase which I don't understand what it is or how it gets populated.)

I don't know if we have thoughts about whether it should be the source or dest but for now I'll just kill this variable.